src/reset-password.scss

```diff
@@ -0,0 +1,35 @@
+@use '~@zero-tech/zui/styles/theme' as theme;
+
+.reset-password-main {
+  display: flex;
+  flex-direction: column;
+  padding: 24px 0;
+  box-sizing: border-box;
+
+  justify-content: space-between;
+  align-content: flex-start;
+  align-items: center;
+
+  background: linear-gradient(0deg, rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0.75)),
+    radial-gradient(66.52% 66.52% at 0% 0%, #26072d 0%, rgba(38, 7, 45, 0) 100%),
+    radial-gradient(93.63% 93.63% at 100% 0%, #9d0097 0%, rgba(157, 0, 151, 0) 100%),
+    radial-gradient(91.27% 91.27% at 112.3% 83.55%, #52a9ff 0%, rgba(82, 169, 255, 0) 100%),
+    radial-gradient(65.88% 65.88% at -1.79% 99.21%, #4f0895 0%, rgba(79, 8, 149, 0) 100%), #290634;
+
+  position: absolute;
+  overflow-y: auto;
+  overflow-x: hidden;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  z-index: 10;
+
+  > * {
+    flex-shrink: 0;
+  }
```

Comment:
Correct. I think this should be handled in another PR though as per previous guidance on refactoring changes out of scope of task.

Reply:
I am not sure if i follow, would like more clarity on this though (just to be sure). I agree that we should have separate PR's for refactoring the code we're touching. But in this particular case we're "adding" a new file, no? If we import the scss from the `invite.scss` here, and re-use it, is that considered into "refactoring" as well? 🤔 

just a doubt, not a big deal anyways. we can absolutely do it in a follow up PR

Reply:
Yeah I think you are also right here. I decided not to use the `invites.scss` file for this PR as really we would want to rename `invites.scss` if we will be re-using it elsewhere - or move it to another folder etc. So for review purposes I thought best to add another in relation to the reset-password to prevent making changes not directly related to `reset-password` i.e. the invite files. But maybe I should have just imported the `invites.scss` file instead, that also would have been fairly simple to follow!